test/integration/src/main/java/io/pravega/test/integration/selftest/Event.java

```diff
@@ -0,0 +1,223 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.pravega.test.integration.selftest;
+
+import com.google.common.base.Preconditions;
+import io.pravega.common.io.StreamHelpers;
+import io.pravega.common.util.ArrayView;
+import io.pravega.common.util.BitConverter;
+import io.pravega.common.util.ByteArraySegment;
+import io.pravega.test.integration.selftest.adapters.TruncateableArray;
+import java.io.IOException;
+import lombok.Getter;
+import lombok.SneakyThrows;
+
+/**
+ * Represents an Event with a Routing Key and payload.
+ */
+public class Event {
+    //region Members
+
+    private static final int PREFIX_LENGTH = Integer.BYTES;
+    private static final int OWNER_ID_LENGTH = Integer.BYTES;
+    private static final int ROUTING_KEY_LENGTH = Integer.BYTES;
+    private static final int SEQUENCE_LENGTH = Integer.BYTES;
+    private static final int START_TIME_LENGTH = Long.BYTES;
+    private static final int LENGTH_LENGTH = Integer.BYTES;
+    static final int HEADER_LENGTH = PREFIX_LENGTH + OWNER_ID_LENGTH + ROUTING_KEY_LENGTH + SEQUENCE_LENGTH + START_TIME_LENGTH + LENGTH_LENGTH;
+    private static final int PREFIX = (int) Math.pow(Math.E, 20);
```

Comment:
What? I wanted to have a specific arbitrary number that denotes the start of every event. If I get some byte stream back that doesn't start with that, then I know this can't be an event.